lightning/src/ln/msgs.rs

```diff
@@ -68,6 +68,8 @@ pub enum DecodeError {
         io::ErrorKind),
 	/// The message included zlib-compressed values, which we don't support.
 	UnsupportedCompression,
+	/// The message included a TLV with an unknown type.
+	UnknownTLV(u64),
```

Comment:
IIUC this variant is never created in this PR? Maybe it can be added later if needed when its crated?

Reply:
Yeah, true. I think I left it behind when I added that out of scope stuff. Will remove 🙂